qa/TL0_multigpu/test_body.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash -e
+
+test_nose() {
+  # placeholder function
+  :
+}
+
+test_py() {
+  # placeholder function
+  :
+}
+
+test_gtest() {
+    for BINNAME in \
+    "dali_core_test.bin" \
+    "dali_kernel_test.bin" \
+    "dali_test.bin" \
+    "dali_operator_test.bin"
+  do
+    for DIRNAME in \
+      "../../build/dali/python/nvidia/dali" \
+      "$(python -c 'import os; from nvidia import dali; print(os.path.dirname(dali.__file__))' 2>/dev/null || echo '')"
+    do
+        if [ -x "$DIRNAME/test/$BINNAME" ]; then
+            FULLPATH="$DIRNAME/test/$BINNAME"
+            break
+        fi
+    done
+
+    if [[ -z "$FULLPATH" ]]; then
+        echo "ERROR: $BINNAME not found"
+        exit 1
+    fi
+
+    "$FULLPATH" --gtest_filter="*MultiDevice*:*SyncCrossDevice*:*FreeOnAnotherDevice*:*GetResource_Device_RangeCheck*"
```

Comment:
Just an idea (maybe for followup): Come up with a single prefix/pattern to refer to C++ multi-gpu tests.

Reply:
Done